patches/@changesets+apply-release-plan+6.1.1.patch

```diff
@@ -0,0 +1,114 @@
+       depVersionRange: versionRange,
+-      depType: dependencyVersionRange ? "dependencies" : "peerDependencies"
++      depType: dependencyVersionRange ? "dependencies" : devDependencyVersionRange ? "devDependencies" : "peerDependencies"
+     }, {
+       minReleaseType: updateInternalDependencies,
+       onlyUpdatePeerDependentsWhenOutOfRange: onlyUpdatePeerDependentsWhenOutOfRange
+diff --git a/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.esm.js b/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.esm.js
+index 86430fd..8526e60 100644
+--- a/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.esm.js
++++ b/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.esm.js
+@@ -201,17 +201,19 @@ async function getChangelogEntry(release, releases, changesets, changelogFuncs,
+     }
+   });
+   let dependentReleases = releases.filter(rel => {
+-    var _release$packageJson$, _release$packageJson$2;
++    var _release$packageJson$, _release$packageJson$2, _release$packageJson$3;
+ 
+     const dependencyVersionRange = (_release$packageJson$ = release.packageJson.dependencies) === null || _release$packageJson$ === void 0 ? void 0 : _release$packageJson$[rel.name];
+-    const peerDependencyVersionRange = (_release$packageJson$2 = release.packageJson.peerDependencies) === null || _release$packageJson$2 === void 0 ? void 0 : _release$packageJson$2[rel.name];
+-    const versionRange = dependencyVersionRange || peerDependencyVersionRange;
++    const devDependencyVersionRange = release.packageJson.name.startsWith("block-template-" // @todo use option from .changeset/config.json instead
++    ) ? (_release$packageJson$2 = release.packageJson.devDependencies) === null || _release$packageJson$2 === void 0 ? void 0 : _release$packageJson$2[rel.name] : undefined;
++    const peerDependencyVersionRange = (_release$packageJson$3 = release.packageJson.peerDependencies) === null || _release$packageJson$3 === void 0 ? void 0 : _release$packageJson$3[rel.name];
++    const versionRange = dependencyVersionRange || devDependencyVersionRange || peerDependencyVersionRange;
+     return versionRange && shouldUpdateDependencyBasedOnConfig({
+       type: rel.type,
+       version: rel.newVersion
+     }, {
+       depVersionRange: versionRange,
+-      depType: dependencyVersionRange ? "dependencies" : "peerDependencies"
++      depType: dependencyVersionRange ? "dependencies" : devDependencyVersionRange ? "devDependencies" : "peerDependencies"
+     }, {
+       minReleaseType: updateInternalDependencies,
+       onlyUpdatePeerDependentsWhenOutOfRange
+diff --git a/node_modules/@changesets/apply-release-plan/src/get-changelog-entry.ts b/node_modules/@changesets/apply-release-plan/src/get-changelog-entry.ts
+index 1f07982..cb3c43f 100644
+--- a/node_modules/@changesets/apply-release-plan/src/get-changelog-entry.ts
++++ b/node_modules/@changesets/apply-release-plan/src/get-changelog-entry.ts
```

Comment:
↑ autogenerated using `yarn build` in [Changesets monorepo](https://github.com/changesets/changesets)
↓ written by hand

Reply:
Should we add a comment into the patch which explains this, in case someone else other than you has to recreate it when updating `changesets`?